### DISCUSSION TITLE
3d printable enclosures

### DISCUSSION THREAD

**Discussion by Ramblurr** [TheRealWaldo/AMG8833-ESPHOME#13] [Ideas]
Has anyone deployed these in their home with a 3d printed enclosure? Would you be willing to share the STL files?

**Comment by TheRealWaldo** [comment]
@Ramblurr I posted cases that I used at:

https://github.com/TheRealWaldo/thermal/discussions/177#discussioncomment-4212228

Please let us know if you make any modifications or find a better one!
